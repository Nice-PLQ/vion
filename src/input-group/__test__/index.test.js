import { mount } from '@vue/test-utils';
import VInputGroup from '../input-group.vue';
import VInput from '../../input/input.vue';

describe('InputGroup', () => {
  test('渲染', () => {
    const wrapper = mount({
      template: `
        <v-input-group space="16px" label-align="center" label-width="70px">
          <v-input label="公司" placeholder="请输入公司" />
          <v-input label="职业" placeholder="请输入职业" />
          <v-input label="邮箱" placeholder="请输入邮箱" />
      </v-input-group>
      `
    }, {
      global: {
        components: { VInputGroup, VInput }
      }
    });
    expect(wrapper.element).toMatchSnapshot();
    expect(wrapper.findAllComponents('.v-input').length).toBe(3);
  });
});
