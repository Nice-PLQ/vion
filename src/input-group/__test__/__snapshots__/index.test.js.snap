// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputGroup 渲染 1`] = `
<div
  class="v-input-group"
  space="16px"
>
  
  <div
    class="v-input"
  >
    <div
      class="v-input__inner"
    >
      <div
        class="v-input__label"
        style="text-align: center; width: 70px;"
      >
        
        <span>
          公司
        </span>
        
      </div>
      <div
        class="v-input__wrap"
      >
        <input
          placeholder="请输入公司"
          type="text"
        />
        <!--v-if-->
      </div>
      <!--v-if-->
    </div>
    <!--v-if-->
  </div>
  <div
    class="v-input"
  >
    <div
      class="v-input__inner"
    >
      <div
        class="v-input__label"
        style="text-align: center; width: 70px;"
      >
        
        <span>
          职业
        </span>
        
      </div>
      <div
        class="v-input__wrap"
      >
        <input
          placeholder="请输入职业"
          type="text"
        />
        <!--v-if-->
      </div>
      <!--v-if-->
    </div>
    <!--v-if-->
  </div>
  <div
    class="v-input"
  >
    <div
      class="v-input__inner"
    >
      <div
        class="v-input__label"
        style="text-align: center; width: 70px;"
      >
        
        <span>
          邮箱
        </span>
        
      </div>
      <div
        class="v-input__wrap"
      >
        <input
          placeholder="请输入邮箱"
          type="text"
        />
        <!--v-if-->
      </div>
      <!--v-if-->
    </div>
    <!--v-if-->
  </div>
  
</div>
`;
